packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -129,7 +129,7 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
                 {(subject) => (
                   <Markdown
                     appearanceKey="notificationSubject"
-                    class="nt-text-start nt-font-medium"
+                    class="nt-text-start nt-font-medium nt-whitespace-pre-wrap [word-break:break-word]"
```

Comment:
fix for long text breaking UI:
![Screenshot 2025-05-07 at 17 02 35](https://github.com/user-attachments/assets/5018b78f-f395-4e1c-88aa-56b0e550997a)